backend.py

```diff
@@ -679,3 +761,17 @@ def get_direction_from_coordinates(start_coordinates, stop_coordinates):
     for direction in list(Direction):
         if direction.coor_delta == delta:
             return direction
+
+
+def cards_as_dict(cards):
+    card_pack = []
+    for card in cards:
+        card_pack.append(card.as_dict())
+    return {"dealt_cards": card_pack}
+
+
+def cards_from_dict(cards):
```

Comment:
co vlastně je ten argument cards? to bude ta přijatá zpráva JSON že?

Reply:
ano, nějak se ty karty musí převést zpátky na objekty. Ale nefunguje to, neumím to zatím správit.